src/data/zmod/basic.lean

```diff
@@ -577,6 +585,50 @@ def units_equiv_coprime {n : ℕ} [fact (0 < n)] :
   left_inv := λ ⟨_, _, _, _⟩, units.ext (nat_cast_zmod_val _),
   right_inv := λ ⟨_, _⟩, by simp }
 
+/-- The **Chinese remainder theorem**. For a pair of coprime natural numbers, `m` and `n`,
+  the rings `zmod (m * n)` and `zmod m × zmod n` are isomorphic. -/
```

Comment:
```suggestion
/-- The **Chinese remainder theorem**. For a pair of coprime natural numbers, `m` and `n`,
  the rings `zmod (m * n)` and `zmod m × zmod n` are isomorphic. 

See `ideal.quotient_inf_ring_equiv_pi_quotient` for the Chinese remainder theorem for ideals in any ring.
-/
```

Reply:
is it not worth proving this from that? or at least making the explicit `hom` a separate definition, like the `ideal` example does

Reply:
Isn't the explicit `hom` just `zmod.cast_hom`? I think it is possible but not worth the effort to hook up the two phrasings to each other, in particular because this is computable and has better defeqs than the general version.